cookbook/form/create_form_type_extension.rst

```diff
@@ -12,18 +12,10 @@ But sometimes, you don't really need to add new field types - you want
 to add features on top of existing types. This is where form type
 extensions come in.
 
-Form type extensions have 2 main use-cases:
-
-#. You want to add a **generic feature to several types** (such as
-   adding a "help" text to every field type);
-#. You want to add a **specific feature to a single type** (such
-   as adding a "download" feature to the "file" field type).
-
```

Comment:
I agree this was a great way to explain Foem Type Extensions.

I restored them but changed the example. I now use ""adding a "help" text to every "text" or "choice"-like type"". This seem to be closer to an actual use case but also more truthful regarding what can be done.